components/list/README.md

```diff
@@ -78,6 +78,41 @@ The `d2l-list-item` provides the appropriate `listitem` semantics for children w
 - `selectable` (Boolean): Indicates a checkbox should be rendered for selecting the item
 - `selected` (Boolean): Whether the item is selected
 
+## ListItemMixin
+
+Want to maintain consistency with `d2l-list-item` but need more modularity? This mixin is for you! This mixin allows you to make a component into a list item without requiring custom styling. All of the properties and functionality from `d2l-list-item` will be added to your new component.
+
+### How to use
+
+Import
+```javascript
+import { ListItemMixin } from './list-item-mixin.js';
+
+class ListItem extends ListItemMixin(LitElement) {
+...
+```
+
+How add the styles:
+```javascript
+static get styles() {
+	return [ super.styles ];
+}
+```
+
+How to render the list item:
+```javascript
+render() {
+	return this._renderListItem({
+		illustration: html`[Image HTML here]`,
+		content: html`[Content here such as d2l-list-item-content]`,
+		actions: html`actions here`
+	});
+}
+```
+Where the parameters correspond to the slots of `d2l-list-item`:
+- illustration (TemplateResult):  Provide a illustration for your list item.
+- content (TemplateResult): Core content of the list item. You can use content components such as `d2l-list-item-content` here. (This corresponds to the default slot in `d2l-list-item`.
```

Comment:
Suggest making this a little more succinct.  "Core content of the list item, such as a `d2l-list-item-content` element."